src/components/calcite-radio-button/calcite-radio-button.tsx

```diff
@@ -126,6 +126,8 @@ export class CalciteRadioButton {
   //
   //--------------------------------------------------------------------------
 
+  private documentRoot: any;
```

Comment:
☝️ I'd like to type this properly, but neither `Node` or `HTMLElement` seem to work right.  Either keeps saying that certain properties are missing... :/

Reply:
The object returned by `getRootNode` extends `Node`, so that type is correct. If the base type you're expecting is `HTMLElement`, you can type it as so and just cast it when you assign the value:

```tsx
this.documentRoot = this.el.getRootNode() as HTMLElement;
```